backend/apps/owasp/graphql/nodes/project.py

```
@@ -53,13 +59,25 @@ def resolve_languages(self, info):
         """Resolve languages."""
         return self.idx_languages
 
-    def resolve_recent_issues(self, info):
-        """Resolve recent issues."""
-        return self.issues.select_related("author").order_by("-created_at")[:RECENT_ISSUES_LIMIT]
+    def resolve_recent_issues(root, info, limit=RECENT_ISSUES_LIMIT, distinct=False):  
```

Ruff (0.8.2)

First argument of a method should be named `self`

Rename `root` to `self`

(N805)